pkgs/misc/vim-plugins/vim-plugin-names

```diff
@@ -460,6 +460,7 @@ peitalin/vim-jsx-typescript
 peterbjorgensen/sved
 peterhoeg/vim-qml
 phaazon/hop.nvim
+phaazon/hop.nvim
```

Comment:
This plugin already is in nix the line above.

Reply:
my bad, I should have checked here instead of searching on search.nixos.org